### ISSUE TITLE
Doesn't prevent new windows from opening (e.g. link with target="_blank")

### ISSUE THREAD

**Issue by matrey** [rosshinkley/nightmare-navigation-lock#2]
Hi
The code doesn't prevent new windows from opening (e.g. when clicking on a link with target=_blank).
Seems like it could be supported easily by using `preventDefault` on the `new-window` event (same concept as with `will-navigate`)

**Comment by rosshinkley** [comment]
Fair point.  I'll look into it, or if you've got time, a PR would be welcome. :)
